boards/arm/sam4s_xplained/ext_ram.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) Justin Watson 2017
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <init.h>
+#include <nanokernel.h>
+#include <pinmux.h>
+#include <soc.h>
+#include <sys_io.h>
+
+/* Static Memory Controller (SMC) */
```

Comment:
Yes, currently there is none. The arch/arm/soc/atmel_sam/sam4s location is also worth considering. I guess it depends if we need any Kconfig options associated with the SMC driver. Maybe one of the maintainers will have some good tips on the naming/location.